sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy

```diff
@@ -62,6 +63,7 @@ class DependencyManager {
     private final NotificationHandler notificationHandler
 
     private LoadProjectsDataSource loadProjectsDataSource
+    private LastChangeDateDataSource lastChangeDataSource
```

Comment:
You have both terms in your code: `LatestChange` and `LastChange` I would prefer `LastChanged`. In any way, please unify your language here a bit 😄  as to avoid confusion later on